lib/content.js

```diff
@@ -0,0 +1,80 @@
+const request = require('request-promise-native');
+const { simplePost, populateSimplePost } = require('./content-structure');
+
+module.exports = {
+  createNewEntry,
+  getLatestVersion,
+  updateEntry,
+  publishEntry
+};
+
+// Promise<[createdEntryId: int, createdEntryVersion: int]>
+async function createNewEntry ({ cmaToken, spaceId }) {
```

Comment:
Why not use the SDK for this?

Reply:
The SDK is quite big & complex, doesn't worth to bring it in for one request I think

Reply:
The SDK might be too big and complex, but that complexity is hidden behind an interface that is similar to what we have in this PR so I would re-write the similar functionality because it the SDK is big/complex. 
Plus, users are more likely to use the SDK to fetch their data, so this makes our tests closer to real life experience of users interacting w/ Contentful.

Reply:
Specifically `create` or all ?

Reply:
For everything that the SDK already does.